src/OrbitClient.php

```diff
@@ -201,6 +202,8 @@ private function getRequestHeaders(array $options)
         }
         if ($this->options['env'] === 'stage') {
             $headers['x-forge-environment'] = 'stage';
+        }
+        if (in_array($this->options['env'], $envs)) {
```

Comment:
I think you can inline $envs directly here as it's not used elsewhere, or at least rename it to $nonLiveEnvs for example to clarify what we are doing.